reqwest-middleware/Cargo.toml

```diff
@@ -11,16 +11,17 @@ categories = ["web-programming::http-client"]
 readme = "../README.md"
 
 [dependencies]
-anyhow = "1.0.0"
-async-trait = "0.1.51"
-http = "0.2.0"
-reqwest = { version = "0.11.10", default-features = false, features = ["json", "multipart"] }
-serde = "1.0.106"
+anyhow = "1.0.81"
+async-trait = "0.1.79"
+http = "1.1.0"
+reqwest = { version = "0.12.1", default-features = false, features = ["json", "multipart"] }
+serde = "1.0.197"
 task-local-extensions = "0.1.4"
-thiserror = "1.0.21"
+thiserror = "1.0.58"
 
 [dev-dependencies]
+reqwest = { version = "0.12.1", features = ["rustls-tls"] }
```

Comment:
Should we bump this to 0.12.2 since we are updating it anyway?

Reply:
I don't think it matters much as long as we don't use any features from `0.12.2` that aren't found in `0.12.1`. It's normal for crates to specify the minimum compatible version in `Cargo.toml` files because cargo automatically selects the latest compatible version without needing changes to `Cargo.toml` files.

So I'm happy either way.